scripts/shell/bt_nus_shell.py

```diff
@@ -1,84 +1,88 @@
-from pc_ble_driver_py.ble_adapter     import BLEAdapter, BLEAdapterObserver, EvtSync
+HOST = ''   # Symbolic name meaning all available interfaces
+PORT =  int(args.port)
 
-global nrf_sd_ble_api_ver
+# global nrf_sd_ble_api_ver
 nrf_sd_ble_api_ver = config.sd_api_ver_get()
-logger  = logging.getLogger(__name__)
+logger = logging.getLogger(__name__)
 
-class BLE_Serial(BLEDriverObserver, BLEAdapterObserver):
-    BASE_UUID   = BLEUUIDBase([0x6E, 0x40, 0x00, 0x00, 0xB5, 0xA3, 0xF3, 0x93,
+
+class BleSerial(BLEDriverObserver, BLEAdapterObserver):
+    BASE_UUID = BLEUUIDBase([0x6E, 0x40, 0x00, 0x00, 0xB5, 0xA3, 0xF3, 0x93,
                                0xE0, 0xA9, 0xE5, 0x0E, 0x24, 0xDC, 0xCA, 0x9E])
 
-    RX_UUID     = BLEUUID(0x0003, BASE_UUID)
-    TX_UUID     = BLEUUID(0x0002, BASE_UUID)
-    LOCAL_ATT_MTU     = 23
-    def __init__(self, comport, periph_name, s_conn):
-        driver           = BLEDriver(serial_port    = comport,
-                                     baud_rate      = 115200)
-        adapter          = BLEAdapter(driver)
-        self.evt_sync           = EvtSync(['connected', 'disconnected'])
+    RX_UUID = BLEUUID(0x0003, BASE_UUID)
+    TX_UUID = BLEUUID(0x0002, BASE_UUID)
+    LOCAL_ATT_MTU = 23
+
+    def __init__(self, com_port, periph_name, s_conn):
+        driver = BLEDriver(serial_port    = com_port,
+                           baud_rate      = 1000000)
+        adapter = BLEAdapter(driver)
+        self.evt_sync = EvtSync(['connected', 'disconnected'])
         self.target_device_name = periph_name
         self.target_device_addr = 0
-        self.conn_handle        = None
-        self.adapter            = adapter
-        self.notifications_q    = Queue.Queue()
+        self.conn_handle = None
+        self.adapter = adapter
+        self.notifications_q = queue.Queue()
         self.adapter.observer_register(self)
         self.adapter.driver.observer_register(self)
         self.s_conn = s_conn
         self.adapter.driver.open()
-        ble_enable_params = BLEEnableParams(vs_uuid_count      = 10,
-                                            service_changed    = False,
-                                            periph_conn_count  = 0,
-                                            central_conn_count = 1,
-                                            central_sec_count  = 1)
+
         if nrf_sd_ble_api_ver >= 3:
             logger.info("\nBLE: ble_enable with local ATT MTU: {}".format(DFUAdapter.LOCAL_ATT_MTU))
-            ble_enable_params.att_mtu = BLE_Serial.LOCAL_ATT_MTU
 
-        self.adapter.driver.ble_enable(ble_enable_params)
-        self.adapter.driver.ble_vs_uuid_add(BLE_Serial.BASE_UUID)
+        gatt_cfg = BLEConfigConnGatt()
+        gatt_cfg.att_mtu = self.adapter.default_mtu
+        gatt_cfg.tag = CFG_TAG
+        self.adapter.driver.ble_cfg_set(BLEConfig.conn_gatt, gatt_cfg)
+
+        self.adapter.driver.ble_enable()
+        self.adapter.driver.ble_vs_uuid_add(BleSerial.BASE_UUID)
```

Comment:
This seems to be replacing the `BLEEnableParams` object with a new `BLEConfigConnGatt` object. Is that because the underlying API changed when going to python 3? Or is it unrelated to python 3?